packages/esm-ward-app/src/ward-view/ward-view.component.tsx

```diff
@@ -92,16 +92,27 @@ const WardViewByLocation = () => {
       return <WardBed key={bed.uuid} bed={bed} wardPatients={wardPatients} />;
     });
 
+    const patientsInBedsUuids = bedLayouts.flatMap((bedLayout) => bedLayout.patients.map((patient) => patient.uuid));
+    const wardUnassignedPatients = admittedPatients
+      .filter((admittedPatient) => !patientsInBedsUuids.includes(admittedPatient.patient.uuid))
+      .map((admittedPatient) => {
+        // TODO: note that this might not display all the correct data until https://openmrs.atlassian.net/browse/EA-192 is done
+        return (
+          <UnassignedPatient wardPatient={{ ...admittedPatient, admitted: true }} key={admittedPatient.patient.uuid} />
+        );
+      });
+
     return (
       <>
         {wardBeds}
-        {bedLayouts.length == 0 && (
+        {bedLayouts.length == 0 && isBedManagementModuleInstalled && (
```

Comment:
Not sure we really want or need this warning, but we can address in a future ticket if necessary.

Reply:
Yeah, agreed... this was an existing warning I preserved.